clients/tfchain-client-go/impl.go

```diff
@@ -13,20 +16,16 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
-const (
-	// acceptable delay is amount of blocks (in second) that a node can
-	// be behind before we don't accept it. block time is 6 seconds, so
-	// right now we only allow 2 blocks delay
-	acceptableDelay = 2 * 6 * time.Second
+var (
+	ErrInvalidVersion         = fmt.Errorf("invalid version")
+	ErrUnknownVersion         = fmt.Errorf("unknown version")
+	ErrNotFound               = fmt.Errorf("object not found")
+	ErrNoConnectionsAvailable = fmt.Errorf("no healthy connections available")
+	ErrMaxPoolSizeReached     = fmt.Errorf("max pool size reached")
 )
 
-var (
-	//ErrInvalidVersion is returned if version 4bytes is invalid
-	ErrInvalidVersion = fmt.Errorf("invalid version")
-	//ErrUnknownVersion is returned if version number is not supported
-	ErrUnknownVersion = fmt.Errorf("unknown version")
-	//ErrNotFound is returned if an object is not found
-	ErrNotFound = fmt.Errorf("object not found")
+const (
+	AcceptableDelay = 2 * 6 * time.Second
```

Comment:
WHy exported?

Reply:
Indeed shouldn't be exported since it's an internal implementation